.github/workflows/benchmarks.yml

```
@@ -0,0 +1,74 @@
+name: Benchmark
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+  workflow_dispatch:
+
+jobs:
+  benchmark:
+    if: ${{ contains( github.event.pull_request.labels.*.name, 'run-benchmark') && github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' }}
```

```suggestion
    if: ${{ github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch' }}
```

this is running in 3 minutes; we can just run it always.

---

Done.